core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowVersionResource.scala

```diff
@@ -120,6 +121,31 @@ object WorkflowVersionResource {
     workflowVersionDao.update(workflowVersion)
   }
 
+  /*
+   * This function retrieves the content of versions from a specific workflow in a range
+   * @param lowerBound lower bound of the version search range
+   * @param UpperBound upper bound of the search range
+   * @param wid workflow id
+   * @return a list of contents as strings
+   */
+  def isVersionInRangeUnimportant(
+      lowerBound: UInteger,
+      UpperBound: UInteger,
+      wid: UInteger
+  ): Boolean = {
+    if (lowerBound==UpperBound) {
+      return false
```

Comment:
is it false? the function says unImportant, also its description deosn't say the return meaning